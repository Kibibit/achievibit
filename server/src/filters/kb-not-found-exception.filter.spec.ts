import { KbNotFoundExceptionFilter } from '@kb-filters';

describe('NotFoundExceptionFilterFilter', () => {
  it('should be defined', () => {
    expect(new KbNotFoundExceptionFilter('')).toBeDefined();
  });
});
