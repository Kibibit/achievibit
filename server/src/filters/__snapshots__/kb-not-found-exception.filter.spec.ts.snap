// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundExceptionFilterFilter should return HTML file for undefined routes not inside /api/ 1`] = `"app-root/dist/client/index.html"`;

exports[`NotFoundExceptionFilterFilter should return exepction if route starts with /api/ 1`] = `
Object {
  "error": "test title",
  "name": "Error",
  "statusCode": 404,
}
`;
